Calculate/Calculator.cs

```diff
@@ -0,0 +1,74 @@
+using System.Globalization;
+
+namespace Calculate;
+
+public class Calculator
+{
+
+    public IReadOnlyDictionary<char, Func<int, int, int>> MathematicalOperations { get; } = 
+        new Dictionary<char, Func<int, int, int>>()
```

Comment:
```suggestion
    public IReadOnlyDictionary<char, Func<int, int, double>> MathematicalOperations { get; } = 
        new Dictionary<char, Func<int, int, double>>()
```
I would recommend a double being returned would be more clear

Reply:
and accurate